src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,24 @@
+package calculator;
+
+import calculator.util.StringUtils;
+
+public class Calculator {
+    private static final String DELIMITER = " ";
+
+    public double calculate(String input) {
+        if (StringUtils.isBlank(input)) {
+            throw new IllegalArgumentException(ExceptionMessage.INVALID_INPUT_STRING);
+        }
+
+        String[] inputArray = input.split(DELIMITER);
+
+        double firstNumber = StringUtils.getNumber(inputArray[0]);
+        for (int i = 1; i < inputArray.length - 1; i += 2) {
+            double secondNumber = StringUtils.getNumber(inputArray[i + 1]);
+
+            firstNumber = Operation.of(inputArray[i]).calculate(firstNumber, secondNumber);
```

Comment:
```suggestion
            firstNumber = Operation.of(inputArray[i])
                                   .calculate(firstNumber, secondNumber);
```
[자바 코딩 컨벤션[(https://google.github.io/styleguide/javaguide.html)에 따르면 이렇게 한 줄에는 하나의 dot(`.`)만 있도록 코딩하는것을 권유합니다.

\* 자바 코딩 컨벤션에 대한 내용은 step3에 나옵니다!

Reply:
오 이 부분은 생각하지 못했어요. 
줄바꿈 처리하는게 훨씬 보기 좋네요 👍 
다음 스텝에서 참조해서 구현해보겠습니다.